src/views/RejectionReasons.vue

```diff
@@ -240,7 +248,42 @@ export default defineComponent({
       } else {
         showToast(translate("Sequence for rejection reasons updated successfully."))
       }
-    }
+    },
+
+    async updateFulfillmentRejectReasonAssocStatus(event: any, reason: any) {
```

Comment:
Toggle may work incorrectly if request to create/update association get failed. Please check.

Reply:
Fixed this case